### ISSUE TITLE
v7.11.0 - Reorder Collections - Where to find the ReOrder Icon as presented in your Documentation?

### ISSUE THREAD

**Issue by YoullNeverSee** [postmanlabs/postman-docs#2005]
Hi Team,
I made it short cause may I overseen somewhat but where can I find the ReOrder Collection Item which you presented in your docs here:
https://learning.getpostman.com/docs/postman/collections/managing-collections/#reorder-collections

I have v7.11.0
and looks this:

![grafik](https://user-images.githubusercontent.com/42741860/69218248-9dce3100-0b70-11ea-8000-e70cdb411a07.png)

I do not want to ReArrange Requests or SubFolders.
I like to ReOrder my Collections or called TopFolders.

Nice to here some suggestions ;o)
Cheers

**Comment by tristandenyer** [comment]
@YoullNeverSee I got word back from the team that we had made the product decision to remove the button. It's good to hear feedback that people want it, and will add it to our roadmap to re-introduce it, but there are no timelines for it yet.

Thanks for helping keep our docs honest! And giving feedback about our product. We really appreciate it.

@SueSmith let's use this ticket to update the docs (remove mention of the button for latest docs—leave in /v5.)
